conf/test_github.config

```diff
@@ -24,7 +24,7 @@ params {
     bed_chunks_polishing = 2
     polishing_on = false
```

Comment:
What stops to turn polishing on? Which you can have ?on the farm but not one the Github CI?

Reply:
The polishing job was running for more than 2 hours before I cancelled it.
It runs ok when tested on farm. However polishing is a low priority feature and we don't really need to use resources for testing it on CI.